cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,35 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
-    
+
+  beforeEach(() => {
+    cy.visit('/login');
+    cy.login();
+  });
+
+
+  it('should login', () => {
+
+    cy.get('#userName-value').contains('Snow228');
+    cy.url().should('equal', 'https://demoqa.com/profile');
+  });
+
+  it('should add book', () => {
+    cy.get('#gotoStore').click();
+    cy.get('#searchBox').should('exist').type('Speaking JavaScript');
+    cy.contains('a', 'Speaking JavaScript').click();
+    cy.get('#description-label').should('exist');
+    cy.get('.text-right > #addNewRecordButton').click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal(`Book added to your collection.`)
   });
+});
 
-  it('', () => {
-    
+  it('should delete book', () => {
+    cy.url('equal','/profile');
+    cy.contains('a', 'Speaking JavaScript').should('exist');
```

Comment:
right now your test depends on the previous one, add another book here and then delete it